src/classes/CW_CampaignAssetTypeTriggerHandler.cls

```diff
@@ -0,0 +1,51 @@
+/**
+* File Name   :    CW_CampaignAssetTypeTriggerHandler.cls
+* Description :    B-108780: Lock the creation of new campaign asset type when the campaign 
+*				   is linked to a roadmap that is "In Approval" or "Approved" 
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 16/04/2014 Cristina P.	Intial
+*
+*/
+public with sharing class CW_CampaignAssetTypeTriggerHandler 
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.